lib/clipboard.coffee

```diff
@@ -1,6 +1,6 @@
 Clipboard =
   _createTextArea: ->
-    textArea = document.createElement("textarea")
+    textArea = DomUtils.createElement("textarea")
```

Comment:
Yes, and it's because the background page doesn't contain `lib/dom_utils.js` in manifest.json.